wall_e/src/resources/cogs/ban.py

```diff
@@ -0,0 +1,344 @@
+
+        # update ban_list and db
+        self.ban_list.append(member.id)
+        await BannedUsers.insert_ban(username, member.id)
+        await BanRecords.insert_record(username, member.id, mod, mod_id, date, reason)
+
+        # unban
+        await self.bot.guilds[0].unban(member)
+        logger.info(f"[Ban intercept()] ban for {username} moved into db and guild ban was removed")
+
+        # report to council
+        e_obj = discord.Embed(title="Ban Hammer Deployed",
+                              colour=discord.Color.red())
+
+        e_obj.add_field(name="Banned User", value=f"**{username}**", inline=True)
+        e_obj.add_field(name="Moderator", value=f"**{mod}**", inline=True)
+        e_obj.add_field(name="Reason", value=f"```{reason}```", inline=False)
+        e_obj.add_field(name="Notification DM", value="NOT SENT, DUE TO NO COMMON GUILD\n", inline=False)
+        e_obj.set_footer(text="Intercepted Moderator Action")
+
+        if e_obj:
+            e_obj.timestamp = date.astimezone(pytz.timezone('Canada/Pacific'))
+            await self.mod_channel.send(embed=e_obj)
+        logger.info(f"[Ban ban()] Message sent to mod channel,{self.mod_channel}, of the ban for {username}.")
+
+    @commands.command()
+    async def initban(self, ctx):
+        """Reads in all guild bans into this ban system"""
+
+        logger.info(f"[Ban initban()] Initban command detected from {ctx.author}")
+
+        try:
+            # audit logs contains info about user who did the banning, the timestamp of the ban, and the reason
+            # however audit logs only go back 3 months, so have to read older bans from the bans list
+            audit_logs = await self.bot.guilds[0].audit_logs(action=discord.AuditLogAction.ban).flatten()
+            bans = await self.bot.guilds[0].bans()
+        except Exception as e:
+            logger.info(f'[Ban initban()] error while fetching ban data: {e}')
+            await ctx.send(f"Encountered the following errors: {e}\n**Most likely need view audit log perms.**")
+            return
+
+        logger.info(f"[Ban initban()] retrieved audit log data for ban actions: {audit_logs}")
+        logger.info(f"[Ban initban()] retrieved ban list from guild: {bans}")
+
+        logger.info("[Ban initban()] Starting process to move all guild bans into db")
+        ban_ids = [ban.user.id for ban in bans]
+        ban_data = []
+        audit_users = []
+        for ban in audit_logs:
+            audit_users.append(ban.target.id)
+
+            ban_data.append([ban.target.name+'#'+ban.target.discriminator,
+                             ban.target.id,
+                             ban.user.name+'#'+ban.user.discriminator,
+                             ban.user.id,
+                             pytz.utc.localize(ban.created_at),
+                             ban.reason if ban.reason else 'No Reason Given!'
+                             ])
+
+        for ban in bans:
+            if ban.user.id in audit_users:
```

Comment:
why not just do `if ban.user.id not in audit_users:` instead of having a conditional statement with `continue`?